addons/web/static/src/views/relational_model.js

```diff
@@ -2093,6 +2087,7 @@ export class DynamicGroupList extends DynamicList {
         return {
             ...super.exportState(),
             groups: this.groups,
+            groupsLimit: this.limit,
```

Comment:
well, this is necessary to not have the DEFAULT_LOAD_LIMIT applied when re-loading